src/app/cluster/AddClusterModal.tsx

```diff
@@ -0,0 +1,35 @@
+import {
+  Modal,
+  Button,
+  TextInput,
+  TextContent,
+  TextList,
+  TextListItem,
+  TextArea,
+} from '@patternfly/react-core';
+import React from 'react';
```

Comment:
Please, move this import to first line, to keep consistency.

Reply:
We need to decide if we care about import ordering and if so, codify that in the linter.

Reply:
I don't so much care about import ordering, but React import first is something like `unspoken rule`. If you look at code in other React projects or at least at tutorial React import in most cases is in first place.

Reply:
Ok, fixed